dynamic_stack_decider/src/dynamic_stack_decider/parser.py

```diff
@@ -194,7 +198,7 @@ def _extract_parameters(token):
     return name, parameter_dict, unset_parameters
 
 
-def _create_tree_element(token, parent):
+def _create_tree_element(token, parent, lnr):
```

Comment:
You should add documentation for the parameter lnr here, as it already exists for token and parent in the lines below